scripts/linters/html_linter.py

```diff
@@ -135,6 +137,29 @@ def handle_starttag(self, tag, attrs):
                             tag, line_number))
                     self.error_messages.append(error_message)
 
+                # Check if there are any white spaces around
+                # attribute assignment.
+                attr_val_structure = '{}="{}"' if value_in_quotes else '{}={}'
```

Comment:
Considering "not value_in_quotes" itself an unexpected pattern (and we do report this on line 133) then should we avoid doing "space around =" check for "not value_in_quotes"? wdyt? (That will also simplify the logic, right?)

Reply:
@DubeySandeep So, I wrote this because suppose, in the case `class = dummy`.  I felt like it should throw both the errors, not in double-quotes and there is space around the attribute rather than just one. WDYT?